build.gradle

```diff
@@ -113,7 +116,7 @@ check.dependsOn installApp
 dependencies {
     compile files("${System.properties['java.home']}/../lib/tools.jar")
     compile 'org.apache.commons:commons-math3:3.5'
-    compile 'org.broadinstitute:gatk:4.alpha-14-g49e944d-SNAPSHOT'
+    compile 'org.broadinstitute:gatk:4.alpha-60-g98f01b0-SNAPSHOT'
     compile 'com.opencsv:opencsv:3.4'
     compile 'org.ojalgo:ojalgo:39.0'
     compile ('org.apache.spark:spark-mllib_2.10:1.4.1') {
```

Comment:
You can update this to 1.5.0 if you want since that's now are minimum spark version.  There *shouldn't be any problems* since it's a minor release, but there may be some new stuff you'd want in 1.5.0.